@xen-orchestra/lite/src/components/pool/network/PoolNetworksPifStatus.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <UiInfo class="text-ellipsis" :accent="statusProps.accent">
+    <span class="text-ellipsis">{{ statusProps.text }}</span>
```

Comment:
I'm not sure the ellipsis is necessary here?

Reply:
Indeed an oversight